File: src/main/java/com/sfh/pokeRogueBot/model/browser/gamejson/GameData.java
package com.sfh.pokeRogueBot.model.browser.gamejson;

import com.google.gson.annotations.SerializedName;
import lombok.Data;

import java.util.List;
import java.util.Map;

@Data
public class GameData {

    @SerializedName("gameStats")
    private GameStats gameStats;
    @SerializedName("eggs")
    private List<Object> eggs;
    @SerializedName("starterData")
    private Map<String, StarterDataEntry> starterData;
    @SerializedName("voucherCounts")
    private Map<String, Integer> voucherCounts;
}

File: src/main/kotlin/com/sfh/pokeRogueBot/model/browser/gamejson/GameData.kt
package com.sfh.pokeRogueBot.model.browser.gamejson

import com.google.gson.annotations.SerializedName

data class GameData(
    @SerializedName("gameStats")
    val gameStats: GameStats?,

    @SerializedName("eggs")
    val eggs: List<Any>?,

    @SerializedName("starterData")
    val starterData: Map<String, StarterDataEntry>?,

    @SerializedName("voucherCounts")
    val voucherCounts: Map<String, Int>?
)